aboutme.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About me!</title>
+    <link rel="stylesheet" href="Aboutme.css">
+  </head>
+  <body>
+    <h1>Day 2 assingment About me!</h1>
+    <p><strong> Hello! My name is Adan Rabadan.</strong><br />I'm atteding Code fellows to become a software developer.
+      Before I attended code fellows i was a machinist for aircraft manufactor,<br /> prior to that I served
+      in the united state Air Force for 8 years as a air transpotation specialist.<br />
+      I'm married with 2 kids ages 2 and 6 both are boys and have birthday coimng up.
+      </p>
+      <h2>Skills</h2>
+      <p>Over the course of my life I have learn some useful skill</p>
+      <ul>
+        <li>I can driver any vehicle with wheel and some without wheel.</li>
+        <li>I can operate and program a CNC machine.</li>
+        <li>Problem solving skills.</li>
+        <li>The importants of teamwork.</li>
+      </ul>
+  <script src = "Aboutme.js" ></script>
```

Comment:
this line should be indented one more level, also you can lose the white space before and after the `=` in your src attribute.